backend/Watchdog.Core/Watchdog.Core.BLL/Services/ApplicationService.cs

```diff
@@ -103,5 +103,37 @@ public async Task<ApplicationDto> CreateAppAsync(NewApplicationDto dto)
 
             return _mapper.Map<ApplicationDto>(application);
         }
+
+        public async Task<ApplicationDto> GetApplicationByIdAsync(int appId)
+        {
+            var application = await _context.Applications
+                .Include(p => p.Platform)
+                .FirstOrDefaultAsync(a => a.Id == appId);
```

Comment:
Please throw an error there if we don't have a project with such Id

Reply:
Add exception